config/nginx/default.conf

```diff
@@ -0,0 +1,21 @@
+server {
+	listen 0.0.0.0;
+	return 301 https://$host$request_uri;
+}
+
+server {
+	listen 443 ssl;
+	server_name 0.0.0.0;
+    ssl_certificate      /etc/certs/comp_codes.crt;
+    ssl_certificate_key  /etc/certs/comp_codes.key;
+    ssl_ciphers          HIGH:!aNULL:!MD5;
```

Comment:
Make sure to use the most secure ciphers. Run ssl labs against it to make sure you get an A+ rating.